egs/aspire/s5/local/multi_condition/reverberate_data_dir.sh

```diff
@@ -72,12 +57,10 @@ for file in cmvn.scp feats.scp reco2file_and_channel; do
 done
 
 python local/multi_condition/get_reverberate_parameter_lists.py \
```

Comment:
as you are directly generating the wav.scp rename this python script to reverberate_wavs.py